### PR TITLE
TM-4197 setting filter date to next day

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
When selecting the same day on the date filter for panel meetings, it would not return the panel meetings for that day

### CODE REVIEW

File: talentmap_api/fsbid/services/panel.py [hunk 1]
```diff
@@ -247,8 +247,8 @@ def convert_panel_query(query={}):
     Converts TalentMap query into FSBid query
     '''
 
-    panelDateStart = query.get("panel-date-start")
-    panelDateEnd = query.get("panel-date-end")
+    panel_date_start = query.get("panel-date-start")
+    panel_date_end = query.get("panel-date-end")
```
Comment (M00staff) [inline talentmap_api/fsbid/services/panel.py:251]: 👍

**Comment by M00staff** [comment]
tested on dev and the functionality works for me - ill differ to someone with better python vision to approve the code itself

File: talentmap_api/fsbid/services/panel.py [hunk 2]
```diff
@@ -257,13 +257,14 @@ def convert_panel_query(query={}):
...
 
     try:
-        if panelDateStart and panelDateEnd:
-            startVal = maya.parse(panelDateStart).datetime().strftime("%Y-%m-%d")
-            endVal = maya.parse(panelDateEnd).datetime().strftime("%Y-%m-%d")
-            filters.append({"col": "pmddttm", "com": "GTEQ", "val": startVal, "isDate": True})
-            filters.append({"col": "pmddttm", "com": "LTEQ", "val": endVal, "isDate": True})
+        if panel_date_start and panel_date_end:
+            start_val = maya.parse(panel_date_start).datetime().strftime("%Y-%m-%d")
+            end_val = maya.parse(panel_date_end)
```
Comment (SidelineCory24) [inline talentmap_api/fsbid/services/panel.py:262]: if you want you put everything together for end_val
`(maya.parse(panel_date_end) + timedelta(days=1)).datetime().strftime("%Y-%m-%d")`

**Comment by elizabeth-jimenez** [comment]
> question: is this ep expected to work the same as the ep for EAS where we have the same calendar picker for ted? In dev1 the ted for EAS returns a value if the start & end dates are the same.

very nice catch - so the WS EPs do work the same; check out the EAS date timestamps, they are all 0, so when we set our date filters to beginning of day, it doesn't matter for EAS, but it does for Panels, cause they do have timestamps > 0

**Comment by SidelineCory24** [comment]
> > question: is this ep expected to work the same as the ep for EAS where we have the same calendar picker for ted? In dev1 the ted for EAS returns a value if the start & end dates are the same.
> 
> very nice catch - so the WS EPs do work the same; check out the EAS date timestamps, they are all 0, so when we set our date filters to beginning of day, it doesn't matter for EAS, but it does for Panels, cause they do have timestamps > 0

noted, discussed more offline as well🙏

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!

**Pull request merged**